contracts/token/ERC20/extensions/ERC4626.sol

```diff
@@ -84,6 +85,7 @@ abstract contract ERC4626 is ERC20, IERC4626 {
      * @dev Attempts to fetch the asset decimals. A return value of false indicates that the attempt failed in some way.
      */
     function _tryGetAssetDecimals(IERC20 asset_) private view returns (bool ok, uint8 assetDecimals) {
+        Memory.Pointer ptr = Memory.getFreePointer();
```

Comment:
Should be fixed at the level of the call instead. A compliant ERC is supposed to return a valid uints8 for the decimals